plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -62,6 +64,7 @@ export class PersonState {
         statsd: StatsD | undefined,
         personManager: PersonManager,
         personContainer: LazyPersonContainer,
+        poEEmbraceJoin: boolean,
```

Comment:
As the `PersonState` `constructor` got a new parameter, we need to update the corresponding `it('updates person')` test in `tests/worker/ingestion/event-pipeline/processPersonsStep.test.ts` to either pass this parameter or expect it to be undefined. We are calling a mock anyways so either one should be fine.